docs/05-technical-reference/00-custom-resources/apix-01-apirule.md

```diff
@@ -74,39 +74,106 @@ spec:
-| **spec.service.namespace** | **NO** | Specifies the Namespace of the exposed service. |
-| **spec.service.port** | **NO** | Specifies the communication port of the exposed service. |
-| **spec.rules** | **YES** | Specifies the array of Oathkeeper access rules. |
-| **spec.rules.service** | **NO** | Services definitions at this level have higher precedence than the service definition at the **spec.service** level.|
-| **spec.rules.service.name** | **NO** | Specifies the name of the exposed service. |
-| **spec.rules.service.namespace** | **NO** | Specifies the Namespace of the exposed service. |
-| **spec.rules.service.port** | **NO** | Specifies the communication port of the exposed service. |
-| **spec.rules.path** | **YES** | Specifies the path of the exposed service. |
-| **spec.rules.methods** | **NO** | Specifies the list of HTTP request methods available for **spec.rules.path**. |
-| **spec.rules.mutators** | **NO** | Specifies the array of [Oathkeeper mutators](https://www.ory.sh/docs/next/oathkeeper/pipeline/mutator). |
-| **spec.rules.accessStrategies** | **YES** | Specifies the array of [Oathkeeper authenticators](https://www.ory.sh/docs/next/oathkeeper/pipeline/authn). The supported authenticators are `oauth2_introspection`, `jwt`, `noop`, `allow`. |
+This following tables list all the possible parameters of a given resource together with their descriptions:
 
 >**CAUTION:** If `service` is not defined at **spec.service** level, all defined rules must have `service` defined at **spec.rules.service** level, otherwise the validation fails.
 
-## Additional information
-
-When you fetch an existing APIRule CR, the system adds the **status** section which describes the status of the VirtualService and the Oathkeeper Access Rule created for this CR. This table lists the fields of the **status** section.
-
-| Field   |  Description |
-|:---|:---|
-| **status.apiRuleStatus** | Status code describing the APIRule CR. |
-| **status.virtualServiceStatus.code** | Status code describing the VirtualService. |
-| **status.virtualService.desc** | Current state of the VirtualService. |
-| **status.accessRuleStatus.code** | Status code describing the Oathkeeper Rule. |
-| **status.accessRuleStatus.desc** | Current state of the Oathkeeper Rule. |
+<!-- TABLE-START -->
+### APIRule.gateway.kyma-project.io/v1beta1
+
+**Spec:**
+
+| Parameter | Type | Description |
+| ---- | ----------- | ---- |
+| **gateway** (required) | string | Specifies the Istio Gateway to be used. |
+| **host** (required) | string | Specifies the URL of the exposed service. |
+| **rules** (required) | \[\]object | Represents the the array of Oathkeeper access rules to be applied. |
+| **rules.&#x200b;accessStrategies** (required) | \[\]object | Specifies the list of access strategies. The supported access strategies are [Oathkeeper](https://www.ory.sh/docs/oathkeeper/pipeline/authn) `oauth2_introspection`, `jwt`, `noop`, and `allow`. |
```

Comment:
The supported ones are all of those listed in https://www.ory.sh/docs/oathkeeper/pipeline/authn